tests/fermi/test_fermionic.py

```diff
@@ -141,6 +161,22 @@ def test_to_mat_error(self):
         with pytest.raises(ValueError, match="n_orbitals cannot be smaller than 2"):
             fw1.to_mat(n_orbitals=1)
 
+    tup_fw_dag = (
+        (fw1, fw1_dag),
+        (fw2, fw2_dag),
+        (fw3, fw3_dag),
+        (fw4, fw4_dag),
+        (fw5, fw5_dag),
+        (fw6, fw6_dag),
+        (fw7, fw7_dag),
+    )
+
+    @pytest.mark.parametrize("fw, fw_dag", tup_fw_dag)
+    def test_adjoint(self, fw, fw_dag):
+        assert fw.adjoint() == fw_dag
+        assert FermiA(0).adjoint() == FermiC(0)
+        assert FermiC(0).adjoint() == FermiA(0)
```

Comment:
Do we need this? Seems like it's constant across all the parameters. Could add this as another test case or just delete it altogether.

Reply:
It doesn't seem necessary, but I added it because the code coverage report complained about these methods not having tests.

Reply:
Ah I see. I would just write this out as a separate test e.g. test_adjoint_fermiac instead.

Reply:
Oh I saw your changes, that works as well.